modules/programs/firefox.nix

```diff
@@ -757,6 +765,19 @@ in {
       }
 
       (mkNoDuplicateAssertion cfg.profiles "profile")
+
+      (let
+        assertProfile = profile:
+          profile.userChrome != ""
+          -> (profile.settings."toolkit.legacyUserProfileCustomizations.stylesheets"
+            != false);
+      in {
+        assertion = any assertProfile (attrValues cfg.profiles);
+        message = ''
+          'userChrome' won't work with
+          'settings."toolkit.legacyUserProfileCustomizations.stylesheets"' set to false.
```

Comment:
```suggestion
          {option}`settings."toolkit.legacyUserProfileCustomizations.stylesheets"` set to false.
```

Reply:
I also added `{option}` to `userChrome`. Not sure if it works? I don't know how to check how it looks like rendered.

Reply:
At least here, it's not being handled at all. .-.
https://home-manager-options.extranix.com/?query=services.polybar.config&release=master

Reply:
I don't think it makes sense to use `{option}` for assertions. I'm not 100% sure how it's handled for the docs, but assertions don't get rendered like the docs do. They just get dumped out verbatim on the terminal.